src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/ObTabletOpResult.java

```diff
@@ -0,0 +1,4 @@
+package com.alipay.oceanbase.rpc.protocol.payload.impl.execute;
+
+public class ObTabletOpResult extends ObTableBatchOperationResult {
```

Comment:
```suggestion
public class ObTableTabletOpResult extends ObTableBatchOperationResult {
```

Reply:
Since you use ObTable as the prefix of struct/class in other structures, we should keep all of our implementation in the same style.

Reply:
done